### ISSUE TITLE
keyboard shortcut help text

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#253]
Looking at the real en.wikipedia.org the alt texts that show keyboard shortcuts show the key with a prefix

For example hovering over the 'Edit' button on a page show the following 'help'
`Edit this page [alt-shift-e]`
Whereas on xowa (gui or http-server)
`You can edit this page. Please use the preview button before saving [e]`

Apart from the fact that the text is different the keyboard shortcut includes `alt-shift-`

Suggested change in my 'alt-shift' branch

**Comment by gnosygnu** [comment]
Thanks for the details as well as the branch.

I took a look at https://github.com/desb42/xowa/commit/7b42d26c7f2ce110036c2b22daf0908f91231b1c and this is definitely possible. However, I think it also needs to have some platform-specific logic, since Mac OS X will show "ctrl-option-e". Let me check the MediaWiki code later to see how they do it.

Also, the "Edit this page" "message" comes from tooltip-ca-edit in the MediaWiki package at "\languages\i18n\en.json". I'll have to update the corresponding XOWA entry at "\xowa\bin\any\xowa\cfg\lang\core\en.gfs" More likely than not, I'll probably have to refresh all these messages as others may have changed.

**Comment by desb42** [comment]
I must admit I know nothing (or at least next to nothing) about Macs.
It is a bit of a pain keeping up with MediaWiki (and unfortunately mostly your pain) - but I do like the end result

**Comment by gnosygnu** [comment]
Yup, being lock in step with MediaWiki is more than I bargained for. That said, appreciate your sentiment on the end result. Thanks!

**Comment by desb42** [comment]
Have just reviewed this, I notice that the alt-shift is being added by javascript logic (or it is now)

Over the time since this issue was raised, I have significantly changed my version of the javascript logic to enable quite a number of features in other wikis

This enables me to implement the short cuts relatively easily (and remove it from the java code)
